content/_code-samples/send-a-memo/js/send-a-memo.js

```diff
@@ -0,0 +1,58 @@
+if (typeof module !== "undefined") {
+    // Use var here because const/let are block-scoped to the if statement.
+    var xrpl = require('xrpl')
+  }
+  // Validate and send a Memo
+  // https://xrpl.org/transaction-common-fields.html#memos-field
+  
+  async function main() {
+    // Connect to a testnet node
+    console.log("Connecting to Testnet...")
+    const client = new xrpl.Client('wss://s.altnet.rippletest.net:51233')
+    await client.connect()
+  
+    // Get account credentials from the Testnet Faucet
+    console.log("Requesting addresses from the Testnet faucet...")
+    const { wallet, balance } = await client.fundWallet()
+    
+    console.log("Account: ", wallet.address)
+    console.log("   Seed: ", wallet.seed)
+
+    // Enter memo data to insert into a transaction
+    const MemoData = Buffer.from("TEXT 123 -+=", 'utf8').toString('hex')
+    const MemoType = Buffer.from("Text", 'utf8').toString('hex')
+    const MemoFormat = Buffer.from("text/csv", 'utf8').toString('hex')
```

Comment:
There's a utility function for doing string to hex conversions in xrpl.js called `convertStringToHex()` - [source](https://github.com/XRPLF/xrpl.js/blob/76b73e16a97e1a371261b462ee1a24f1c01dbb0c/packages/xrpl/src/utils/stringConversion.ts#L8-L10)

Reply:
Thanks!